src/SME.SGP.Api/Controllers/EncaminhamentoAEEController.cs

```diff
@@ -140,7 +140,7 @@ public async Task<IActionResult> EncerrarEncaminhamento([FromBody] EncarramentoE
         [HttpPost("enviar-analise/{encaminhamentoId}")]
         [ProducesResponseType(typeof(RetornoBaseDto), 200)]
         [ProducesResponseType(typeof(RetornoBaseDto), 500)]
-        [Permissao(Permissao.AEE_A, Policy = "Bearer")]
+        //[Permissao(Permissao.AEE_A, Policy = "Bearer")]
```

Comment:
não remover permissionamento do endpoint
```suggestion
        [Permissao(Permissao.AEE_A, Policy = "Bearer")]
```